redis/commands/core.py

```diff
@@ -1490,16 +1490,28 @@ def exists(self, *names: KeyT) -> ResponseT:
 
     __contains__ = exists
 
-    def expire(self, name: KeyT, time: ExpiryT) -> ResponseT:
+    def expire(self, name: KeyT, time: ExpiryT, option: str = None) -> ResponseT:
```

Comment:
@dogukanteber Sorry, did I miss something? I swear I saw this PR origially including these optoins as items in the function header (i.e separate NX, XX, etc).  If not... what do you think about doing that? It would match our other functions!

Reply:
I have pushed some changes @chayim. Does it match with other functions?